### PR TITLE
Fix clean up ci pipeline

### PR DESCRIPTION
**Pull request by 123MwanjeMike**
**_WHAT DOES THIS PR DO?_**
- Cleans up the cicd pipeline to remove unsed steps. These steps always caused errors that were not factual whenever micros were being deployed
  - [delete old yaml files](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/7900edfaa99922f57cced2b399ef043a3abf5cd6)\
  - [remove staging redundant cicd steps](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/7beea1483e3e0d00f93c2042855d5aa635499f9d)
  - [remove PRODUCTION redundant cicd steps](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/696d3819d3943b47ca8f74f588fea48965946c63)

**_WHAT ISSUES ARE RELATED TO THIS PR?_**

**_HOW DO I TEST OUT THIS PR?_**

**_WHICH ENDPOINTS SHOULD BE READY FOR TESTING?:_** 

**_IS THERE ANY JENKINS CONSOLE LINK FOR CODE COVERAGE AND BUILD INFO?_**

**_ARE THERE ANY RELATED PRs?_**
- https://github.com/airqo-platform/AirQo-frontend/pull/1045 in the frontend

### CODE REVIEW

File: k8s/auth-service/prod-auth-api.yaml [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-    type: RollingUpdate
-    rollingUpdate:
-      maxSurge: 1
-      maxUnavailable: 1
-  minReadySeconds: 5
-  template:
-    metadata:
-      labels:
-        app: auth-api
-    spec:
```
Comment (123MwanjeMike) [inline k8s/auth-service/prod-auth-api.yaml:25]: [delete old yaml files](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/7900edfaa99922f57cced2b399ef043a3abf5cd6)

File: .github/workflows/deploy-apis-to-staging.yml [hunk 2]
```diff
@@ -375,12 +375,6 @@ jobs:
           gcloud secrets versions access latest --secret="sta-key-auth-service-firebase-admin-sdk" > firebase_admin_sdk.json
           kubectl create configmap --dry-run=client -o yaml --from-file=firebase_admin_sdk.json stage-auth-api-config-files | kubectl replace -f - -n staging
 
-      - name: Deploy to K8S
-        run: |
-          kubectl apply -f k8s/auth-service/stage-auth-api.yaml
-          kubectl set image deployment/airqo-stage-auth-api sta-auth-api=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-auth-api:${{ needs.image-tag.outputs.build_id }} -n staging
-          kubectl annotate deployment/airqo-stage-auth-api kubernetes.io/change-cause="Image updated to ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-stage-auth-api:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n staging
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-staging.yml:382]: [remove staging redundant cicd steps](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/7beea1483e3e0d00f93c2042855d5aa635499f9d)

File: .github/workflows/deploy-apis-to-production.yml [hunk 3]
```diff
@@ -418,12 +418,6 @@ jobs:
           gcloud secrets versions access latest --secret="prod-key-auth-service-firebase-admin-sdk" > firebase_admin_sdk.json
           kubectl create configmap --dry-run=client -o yaml --from-file=firebase_admin_sdk.json prod-auth-api-config-files | kubectl replace -f - -n production
 
-      - name: Deploy to K8S
-        run: |
-          kubectl apply -f k8s/auth-service/prod-auth-api.yaml
-          kubectl set image deployment/airqo-auth-api auth-api=${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-auth-api:${{ needs.image-tag.outputs.build_id }} -n production
-          kubectl annotate deployment/airqo-auth-api kubernetes.io/change-cause="Image updated to ${{ env.REGISTRY_URL }}/${{ env.PROJECT_ID }}/airqo-auth-api:${{ needs.image-tag.outputs.build_id }} on ${{ needs.image-tag.outputs.datetime }}" -n production
```
Comment (123MwanjeMike) [inline .github/workflows/deploy-apis-to-production.yml:425]: [remove PRODUCTION redundant cicd steps](https://github.com/airqo-platform/AirQo-api/pull/1314/commits/696d3819d3943b47ca8f74f588fea48965946c63)

**Review by Baalmart — COMMENTED** [review]
Hi @123MwanjeMike , please resolve the merge conflicts. Thanks!

**Comment by 123MwanjeMike** [comment]
> Hi @123MwanjeMike , please resolve the merge conflicts. Thanks!

Right away @Baalmart

**Review by NoahNsimbe — APPROVED** [review]
LGTM

**Pull request merged**